tools/docker/start.sh

```diff
@@ -1,14 +1,24 @@
 #!/bin/bash
+set -ex
 
-cd web-platform-tests
-git pull --depth=50
+REPO=${1:-origin}
+BRANCH=${2:-master}
+REV=${3:-master}
+BROWSER=${4:-all}
+
+cd ~/web-platform-tests
+git fetch ${REPO} ${BRANCH} -n -q --depth=50
```

Comment:
The `--depth=50` still seems like a bit of a liability, making it impossible to restart a failed job if it failed more than 50 commits ago, and if we merge in other repos (like csswg-test) that might be just the previous commit.

How much time does it save to use `--depth=50`?

An alternative, which seems to work, is to make Taskcluster depend on the merge_pr_* tags. Then we could do `git fetch origin merge_pr_10791 --depth=1` here.

At minimum, a comment explaining why `--depth=50` is a good idea would be appreciated :)

Reply:
Yeah, let's call it a "later" problem.